packages/app/src/components/Admin/Security/LocalSecuritySettingContents.jsx

```diff
@@ -30,8 +33,25 @@ class LocalSecuritySettingContents extends React.Component {
     }
   }
 
+  async retrieveIsMailerSetup() {
+    const { adminAppContainer } = this.props;
+    try {
+      await adminAppContainer.retrieveAppSettingsData();
+      this.setState({ isMailerSetup: adminAppContainer.state.isMailerSetup });
+    }
+    catch (err) {
+      toastError(err);
+    }
+  }
+
+  componentDidMount() {
+    this.retrieveIsMailerSetup();
+  }
+
```

Comment:
Ah, sorry 
You don't need to use adminAppContainer and it's better to use `appContainer.config`  instead.

So, please delete this code and refer to `UserInviteModal.jsx`.